.github/workflows/php.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
     strategy:
       matrix:
-        container: [ "php74", "php80", "php81", "php82" ]
+        container: [ "php81", "php82" ]
```

Comment:
Do we need to add 8.3 here?

Reply:
The answer to this question (and to the following ones) is in my comment above: we still don't have a docker image for PHP 8.3